client/templates/home/footer.js

```diff
@@ -0,0 +1,35 @@
+Template.footer.helpers({
+  languageSwitchers: function() {
+    const supportedLanguages = TAPi18n.getLanguages();
+    const currentLanguageCode = TAPi18n.getLanguage();
+    return Object.keys(supportedLanguages).map(function(languageCode){ return {
+      name: supportedLanguages[languageCode].name,
+      attributes: {
+        'data-lang':'',
+        'data-lang-code': languageCode,
+        'data-lang-name': supportedLanguages[languageCode].name,
+        'class': (languageCode === currentLanguageCode ? 'selected_language' : '')
+      }};
+    });
+  }
+});
+
+Template.footer.events({
+  "click [data-lang]": function(event, template){
+    event.preventDefault();
+    const languageCode = event.target.dataset.langCode
+    const languageName = event.target.dataset.langName
+
+    TAPi18n.setLanguage(languageCode)
```

Comment:
I feel like setting the same language again should be a noop. right now I am getting a notification for each click.

Reply:
Makes sense and was intentional, but valid point. I added a guard clause.